docs/model_library/index.rst

```diff
@@ -167,3 +181,14 @@ The constraint described by equation :eq:`eq:sym2` ensures that the largest proj
       :label: eq:sym2 
 
       \sum_{w \in W} \textcolor{red}{x_{wk}} \geq \sum_{w \in W} \textcolor{red}{x_{w(k+1)}} \quad \forall k \in K - \{K_{max}\}
+
+
+**Budget usage**
+
+The constraint defined by equation :eq:`eq:minb` is implemented to ensure that as much of the budget as possible is utilized.
```

Comment:
The constraint only ensures that the budget not utilized is correctly calculated. Suggest rephrase to: "The constraint ensures that the excess amount of budget remaining is correctly calculated" or something similar. Also note on variable notation.